src/AutoUI/index.tsx

```diff
@@ -686,36 +697,28 @@ const getColumnsFromSchema = <T extends AutoUIBaseResource<T>>({
 }): Array<AutoUIEntityPropertyDefinition<T>> =>
 	Object.entries(schema.properties ?? {})
 		.flatMap(([key, val]: [Extract<keyof T, string>, JSONSchema]) => {
-			const refScheme = sieve.getPropertyScheme(val);
+			const refScheme = getPropertyScheme(val);
 			if (!refScheme || refScheme.length <= 1 || typeof val !== 'object') {
 				return [[key, val]];
 			}
-			const entityFilterOnly = sieve.parseDescriptionProperty<string[] | true>(
-				val,
-				'x-filter-only',
-			);
+			const entityFilterOnly = parseDescriptionProperty(val, 'x-filter-only');
 			return refScheme.map((propKey: string) => {
-				const referenceSchema = sieve.generateSchemaFromRefScheme(
-					val,
-					key,
-					propKey,
+				// TODO: replace with the new method
```

Comment:
the new method?

Reply:
left over... there is no new method at the end, I've tried but it was becoming insanely unreadable 😢